README.md

```diff
@@ -1,4 +1,4 @@
-# Quarkus QuteServerPages
+# Quarkus Qute Web
 
 [![Version](https://img.shields.io/maven-central/v/io.quarkiverse.quteserverpages/quarkus-qute-server-pages.svg?label=Maven%20Central)](https://search.maven.org/artifact/io.quarkiverse.quteserverpages/quarkus-qute-server-pages)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
```

Comment:
you need to change the doc here 👇

Reply:
I was waiting for the repo rename. We'll have to do some other stuff afterwards anyway...